coding-standards/git.md

```diff
@@ -49,7 +52,20 @@ Pull Requests (PRs):
 
 -   Create PRs for merging changes into the `main` or `development` branches.
 -   Ensure PRs are reviewed and approved by peers before merging.
--   Provide a detailed description in the PR, explaining the changes and their purpose (why?).
+-   Provide a detailed description in the PR. Use the body to explain what is it, why is it needed and how is it done,
+> \<What is it \>
+> \<Why is it needed\>
+> \<How did we achieve this\>
+-   It is always a best practise to maintain checklist in every PR to make sure everything is submitted properly along with your PR. Example checklist can be,
+> [ ] No build errors
```

Comment:
![image](https://github.com/OsmosysSoftware/dev-standards/assets/16570290/5c523a0b-f5e1-4db4-a8ff-7aec9baed5b7)

This does not look good. KIndly don't add it as a note. Try to format it better.

Reply:
Fixed format issues